emperor/support_files/js/view.js

```diff
@@ -1097,6 +1104,106 @@ DecompositionView.prototype.toggleLabelVisibility = function() {
   this.needsUpdate = true;
 };
 
+
+/**
+ * Set the emissive attribute of the markers
+ *
+ * @param {Bool} emissive Whether the object should be emissive.
+ * @param {Plottable[]} group An array of plottables for which the emissive
+ * attribute. If this object is not provided, all the plottables in the
+ * view will be have the scale set.
+ */
+DecompositionView.prototype.setEmissive = function(emissive, group) {
+  group = group || this.decomp.plottable;
+
+  if (this.decomp.isArrowType()) {
+    throw new Error('Cannot set emissive attribute of arrows');
+  }
+
+  var i = 0, j = 0;
```

Comment:
Is this OK? I mean, these variables are used in the for's below ...

Reply:
Yes, I ended up doing this because we have a couple `for` loops where I would have needed to use `var i` and the linter would have complained that I was declaring each variable multiple times. The solution to this issue is to declare all of these at the top of the scope.